<launch>
    <node pkg="rosbag_record_service" type="record_service_node.py" name="record_service" output="screen">
        <param name="output_folder" value="~/bags" />
<rosparam param="topic_groups">
test1:
  all: False
  regex: qwer
test2:
  all: True
  regex: asdf
</rosparam>
    </node>
</launch>
